### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by vinaysshenoy**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,26 @@
 
 ### Internal
 
+- [In Progress: 22 Sep 2021] Custom Drug Entry Sheet UI Improvements
```
Comment (janhavisinghh) [inline CHANGELOG.md:7]: > Custom Drug Entry Sheet UI Improvements
this is already completed and already in the demo changelog. Can we remove this from here? There must have been some mistake while rebasing somewhere which is why this is here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,26 @@
...
+- [In Progress: 31 Aug 2021] Refactor logic around providing drug frequencies label depending on the country
```
Comment (janhavisinghh) [inline CHANGELOG.md:8]: > Refactor logic around providing drug frequencies label depending on the country

Same with this one. This is already completed and part of the changelog.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,26 @@
...
+- [In Progress: 2 Sep 2021] Add a progress state in `CustomDrugEntrySheet`
```
Comment (janhavisinghh) [inline CHANGELOG.md:9]: > Add a progress state in `CustomDrugEntrySheet`

this is already completed and already in the demo changelog. Can we remove this from here? There must have been some mistake while rebasing somewhere which is why this is here.

File: CHANGELOG.md (hunk 2 above)
Comment (msasikanth) [reply CHANGELOG.md:8]: Removed this in favour of the changelog entries that mention what the refactoring is (providing as a map or injecting into specific screens/view etc.,)

File: CHANGELOG.md (hunk 3 above)
Comment (msasikanth) [reply CHANGELOG.md:9]: I am guessing the drug entry sheet UI improvements is blocked by the final PR that is open @janhavisinghh ?

Comment (janhavisinghh) [reply CHANGELOG.md:9]: Yeah, I'm talking about the progress state changelog

Comment (msasikanth) [reply CHANGELOG.md:9]: Yeah yeah, I am editing the changelog. So checking to see if I should remove the in-progress for the above entry as well.

**Review by msasikanth — APPROVED** [review]
LGTM

**Pull request merged**